packages/docs/developer-resources/integrations/listings.md

```diff
@@ -0,0 +1,107 @@
+
+This guide will also help you find all the necessary information about brand assets, how to integrate with Celo and what useful listing information are made available to you as well as any information about looking for support.
+
+## Celo Brand Assets for Listing
+If you are listing Celo on your exchange, you will probably need access to the Celo Platform brand assets. They can be found [here](https://celo.org/experience/brand#overview).  
+
+Please ensure your use of the Celo Platform assets provided follows the brand policy found [here](https://celo.org/brand-policy).
+
+## How Tos
+### Integrating Celo With Your Infrastructure
+There are several ways to integrate the Celo Platform with your infrastructure.
+
+A general overview of integrations that would be relevant to you listing Celo Platform are shown [here](https://docs.celo.org/developer-guide/overview/integrations/general).
+
+For more specific use-cases for exchanges, please checkout the [Custody and Exchange Integration Guide](https://docs.celo.org/developer-guide/overview/integrations/custody) as well.
+
+## Important Information
+### Celo Native Asset and Celo Dollar
+
+There are two important assets on the Celo network, the Celo native asset (CELO) and the Celo Dollar (cUSD). CELO was called Celo Gold (cGLD) when the contract was deployed, so you will often see references to Celo Gold in the codebase. To learn more about the two, please read [this](https://docs.celo.org/developer-guide/overview/celo-for-eth-devs#the-celo-native-asset-and-the-celo-dollar) section of the docs.
+
+## Resources
+### Address for CELO and cUSD
+The smart contract address for CELO can be found [here](https://explorer.celo.org/address/0x471ece3750da237f93b8e339c536989b8978a438/transactions).
+The smart contract address for cUSD can be found [here](https://explorer.celo.org/address/0x765de816845861e75a25fca122bb6898b8b1282a/transactions).
+
+
+### Useful API endpoints
+The following are useful API endpoints available to you that would help you in your listings of the CELO and cUSD digital assets.
+
+#### CELO and cUSD
+For querying the API on total coins in circulation in CELO and cUSD, the following endpoint will provide you with that:
+```sh
+$ curl  [https://thecelo.com/api/v0.1.js?method=ex_totalcoins](https://thecelo.com/api/v0.1.js?method=ex_totalcoins) {"code":"200","msg":"success","data":{"CELO":608485841.9959723,"cUSD":10250632.56099673}}
+```
+
+##### cUSD Circulating Supply
+Circulating Supply refers to the # of coins that are circulating in the market and in the general public's hands.
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_cusd_circulating
+11353464.550486518
+```
+
+#### CP-DOTO 
+
+For API endpoints useful for listing that follow [CMC requirements](https://docs.google.com/document/d/1S4urpzUnO2t7DmS_1dc4EL4tgnnbTObPYXvDeBnukCg/edit#)
+
+##### Summary
+Summary overview.
+
```

Comment:
Might want to add a bit more of a description (so that visitors don't have to click to CMC doc or in case it's no longer available) e.g. 

For Summary
"Overview of market data for all tickers and all markets"